airflow/ui/package.json

```diff
@@ -42,5 +42,8 @@
     "webpack": "^4",
     "webpack-cli": "^3",
     "webpack-dev-server": "^3"
+  },
+  "resolutions": {
```

Comment:
What's this key for? Not seen this one before.

Reply:
A warning was bubbling up from Chakra UI. They're using a deprecated function from a dependency. `resolutions` let's us swap out their dependency to a slightly older one which doesn't fire the warning. We can remove it when Chakra releases an update.

See:
https://github.com/chakra-ui/chakra-ui/issues/3459#issuecomment-792906739
https://classic.yarnpkg.com/en/docs/selective-version-resolutions/

Reply:
Ohh yeah yarn fix for updating nested dependencies. Gotcha.

We can keep this if it's helpful.

Reply:
Yarn was erroring in the CI build so we're seeing if this caused it.